snappy_pipeline/workflows/somatic_gene_fusion_calling/__init__.py

```diff
@@ -470,6 +470,16 @@ def get_log_file(self, action):
         for key, ext in key_ext:
             yield key, prefix + ext
             yield key + "_md5", prefix + ext + ".md5"
+        prefix = "work/{name}.{{library_name}}/log/".format(name=self.name)
```

Comment:
It doesn't seem like you are using this anywhere, at least not in the wrapper. 
Is it necessary?

Reply:
Nevermind, it is going to `output`.